spec/intl.html

```diff
@@ -1267,281 +1267,273 @@ <h1>Temporal.Calendar.prototype.dateUntil ( _one_, _two_, _options_ )</h1>
-              1. Set _dateOrDateTime_ to ? ToTemporalDate(_dateOrDateTime_).
+            1. Let _temporalDate_ be ? ToTemporalDate(_temporalDateLike_).
             1. If _calendar_.[[Identifier]] is *"iso8601"*, then
               1. Return *undefined*.
-            1. Let _era_ be the result of implementation-defined processing of _dateOrDateTime_ and the value of _calendar_.[[Identifier]].
+            1. Let _era_ be the result of implementation-defined processing of _temporalDate_ and the value of _calendar_.[[Identifier]].
             1. Return _era_.
           </emu-alg>
         </emu-clause>
 
         <emu-clause id="sec-temporal.calendar.prototype.erayear">
-          <h1>Temporal.Calendar.prototype.eraYear ( _dateOrDateTime_ )</h1>
+          <h1>Temporal.Calendar.prototype.eraYear ( _temporalDateLike_ )</h1>
           <p>
-            The `eraYear` method takes one argument _dateOrDateTime_.
+            The `eraYear` method takes one argument _temporalDateLike_.
             The following steps are taken:
           </p>
           <emu-alg>
             1. Let _calendar_ be the *this* value.
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
-            1. If Type(_dateOrDateTime_) is not Object or _dateOrDateTime_ does not have an [[ISOYear]] internal slot, then
-              1. Set _dateOrDateTime_ to ? ToTemporalDate(_dateOrDateTime_).
+            1. Let _temporalDate_ be ? ToTemporalDate(_temporalDateLike_).
             1. If _calendar_.[[Identifier]] is *"iso8601"*, then
               1. Return *undefined*.
-            1. Let _eraYear_ be the result of implementation-defined processing of _dateOrDateTime_ and the value of _calendar_.[[Identifier]].
+            1. Let _eraYear_ be the result of implementation-defined processing of _temporalDate_ and the value of _calendar_.[[Identifier]].
             1. Return 𝔽(_eraYear_).
           </emu-alg>
         </emu-clause>
 
         <emu-clause id="sup-temporal.calendar.prototype.year">
-          <h1>Temporal.Calendar.prototype.year ( _dateOrDateTime_ )</h1>
+          <h1>Temporal.Calendar.prototype.year ( _temporalDateLike_ )</h1>
           <p>This definition supersedes the definition provided in <emu-xref href="#sec-temporal.calendar.prototype.year"></emu-xref>.</p>
           <p>
-            The `year` method takes one argument _dateOrDateTime_.
+            The `year` method takes one argument _temporalDateLike_.
             The following steps are taken:
           </p>
           <emu-alg>
             1. Let _calendar_ be the *this* value.
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
+            1. If Type(_temporalDateLike_) is not Object or _temporalDateLike_ does not have an [[InitializedTemporalDate]] or [[InitializedTemporalYearMonth]] internal slot, then
```

Comment:
Can remove all these checks for InitializedTemporalDate for the same reason we don't do them in the polyfill - ToTemporalDate shortcuts if InitializedTemporalDate already present.

Reply:
True, but I thought the intention would be clearer to the reader if I left them in. Happy to go either way.

Reply:
I'm happy either way too; worth nothing that this makes the spec more brittle to changes, but since the spec shouldn't be seeing many more changes, I'm ambivalent. Your call!

Reply:
It's an editorial change if we want to change it in the future, so I'll resolve this as-is for expediency.